src/Farmer/Common.fs

```diff
@@ -167,6 +167,88 @@ module Route =
             | VirtualNetworkGateway -> "VirtualNetworkGateway"
             | VnetLocal -> "VnetLocal"
 
+module DedicatedHosts =
+    type HostTier =
+        | Standard
+        | Basic
+
+        static member Print(x: HostTier) =
+            match x with
+            | Standard -> "Standard"
+            | Basic -> "Basic"
+
+    type HostSku =
+        | HostSku of string
+
+        static member Print(HostSku x) = x
+        member this.JsonProperties = {| name = HostSku.Print this |}
+
+    type PlatformFaultDomain =
+        | Zero
```

Comment:
Per comment above, fault domains are not limited to 0-3.  Can we make this an integer? Perhaps validate >=0?